src/modules/bank/ajaxifyBank.js

```diff
@@ -1,137 +1,126 @@
 import indexAjaxData from '../ajax/indexAjaxData';
-import jQueryPresent from '../common/jQueryPresent';
+import createDiv from '../common/cElement/createDiv';
+import closestTable from '../common/closestTable';
+import getElementById from '../common/getElementById';
+import getText from '../common/getText';
+import onclick from '../common/onclick';
 import partial from '../common/partial';
-import { defTable, guildSubcmdUrl } from '../support/constants';
+import querySelector from '../common/querySelector';
+import querySelectorArray from '../common/querySelectorArray';
+import setInnerHtml from '../dom/setInnerHtml';
+import setText from '../dom/setText';
+import { guildSubcmdUrl } from '../support/constants';
+import { pcc } from '../support/layout';
 import createDocument from '../system/createDocument';
 
-const statbarGold = '#pH #statbar-gold';
-const statbarGoldTooltip = '#pH #statbar-gold-tooltip-general dd';
-const pccB = '#pCC b';
-const infoMsg = '#pCC #info-msg';
 const withdrawAmount = '#pCC #withdraw_amount';
 const depositAmount = '#pCC #deposit_amount';
-const disabled = 'disabled';
 const inputDepo = '#pCC input[value="Deposit"]';
 
-function doInfoBox(infoBox) { // jQuery
-  const target = $(infoMsg);
-  if (target.length === 0) {
-    $('#pCC').prepend(infoBox.closest(defTable));
+function pccB(doc = document) {
+  return querySelectorArray('#pCC b', doc);
+}
+
+function doInfoBox(infoBox) {
+  const target = getElementById('info-msg');
+  if (target) {
+    closestTable(target).replaceWith(infoBox);
   } else {
-    target.closest(defTable).replaceWith(infoBox.closest(defTable));
+    pcc().prepend(closestTable(infoBox));
   }
 }
 
-function eachGoldValue(doc, index) {
-  return $(statbarGoldTooltip, doc).eq(index).text();
+function updateNodeArray(query, doc) {
+  const newDoc = querySelectorArray(query, doc);
+  querySelectorArray(query).forEach((e, i) => {
+    setText(getText(newDoc[i]), e);
+  });
 }
 
 function doStatBarGold(doc) {
-  $(statbarGold).text($(statbarGold, doc).text());
-  $(statbarGoldTooltip).text(partial(eachGoldValue, doc));
-}
-
-function newStats(doc, balPos, index) {
-  return $(pccB, doc).slice(balPos).eq(index).text();
+  updateNodeArray('#statbar-gold', doc);
+  updateNodeArray('#statbar-gold-tooltip dd', doc);
 }
 
-function doBoldText(doc, balPos) {
-  $(pccB).slice(balPos).text(partial(newStats, doc, balPos));
+function doBoldText(doc) {
+  updateNodeArray('#pCC b', doc);
 }
 
-function disableDepo(depoPos) { // jQuery
-  if ($(pccB).eq(depoPos).text() === '0') {
-    $(inputDepo).prop(disabled, true);
+function disableDepo(depoPos) {
+  if (getText(pccB()[depoPos]) === '0') {
+    querySelector(inputDepo).disabled = true;
   }
 }
 
-function updateDepoAmount(o, doc) { // jQuery
+function updateDepoAmount(o, doc) {
+  const depo = querySelector(depositAmount);
   if (o.data.amount !== '1') {
```

Comment:
bankSettings.data.amount is not being set any more.
when depositing 1 gold into the guild bank which is common when people are working on their medal.
we want to set deposit to 1 instead of the real depo amount.